clio-client/src/main/scala/org/broadinstitute/clio/client/dispatch/DeliverCramExecutor.scala

```diff
@@ -19,19 +19,19 @@ import scala.concurrent.ExecutionContext
   *   1. Writes the cram md5 value to file at the target path
   *   2. Records the workspace name in the metadata for the delivered cram
   */
-class DeliverWgsCramExecutor(deliverCommand: DeliverWgsCram)(
+class DeliverCramExecutor(deliverCommand: DeliverCram)(
```

Comment:
I don't think it'll be possible to use this for either `DeliverWgsCram` or `DeliverCram` without a supertype for the commands.